tabbie2.git/common/models/Tournament.php

```diff
@@ -648,6 +648,15 @@ public function getQuestions($type)
             ->where(["apply_" . $field => 1]);
     }
 
+    /**
+     * @return \yii\db\ActiveQuery
+     */
+    public function getAllQuestions()
```

Comment:
Is there a standard style for code in this project? For example, this file uses spaces while `TournamentController.php` uses tabs.